src/components/BuildingsTable.vue

```diff
@@ -31,6 +31,24 @@ export default class BuildingsTable extends Vue {
   @Prop({ default: false }) showGasUse!: boolean;
 
   @Prop({ default: false }) showElectricityUse!: boolean;
+
+  /** Props from Search component to store field (column) being sorted and direction */
+  @Prop({ default: 'GHGIntensity' }) sortedField!: string;
+  @Prop({ default: 'desc' }) sortedDirection!: string;
+
+  // declares the property emit for TS
+  $emit: any;
```

Comment:
Interesting, I'm a little confused what's wrong here, but I'll need to dig into that more later.

Could you add a todo around this? (e.g. `// TODO: Figure out why Vue `$emit` isn't recognized by Typescript`). You shouldn't have to add this hack, so it's probably something I misconfigured 😅